lib/db/db.js

```diff
@@ -31,15 +35,19 @@ class DB {
   async init() {
     try {
       await this.sequelize.authenticate();
-      this.logger.info('connected to database');
+      const {host, port, database} = this.sequelize.config;
+      const {dialectName} = this.sequelize.connectionManager;
+      this.logger.info(`connected to database. host:${host} port:${port} database:${database} dialect:${dialectName}`);
     } catch (err) {
-      this.logger.error('unable to connect to the database: ', err);
+      this.logger.error('unable to connect to the database', err);
+      throw err;
     }
 
     // sync schemas with the database
+    const options = {logging: this.logger.info};
```

Comment:
I was getting weird (and very verbose) logging output when I tried passing in the logger previously, have you been able to test this at all to see what the output looks like?

Reply:
it’s verbose even with the default console logger (we can reduce it if it’s disturbing).
Now it’s just using the logger’s color, timestamp, etc. Nothing major.

Reply:
OK cool, I thought I was seeing way more output when I set the logger although I did think it was very verbose either way. We can possibly reduce the db logging output in the future.